src/shared/services/payment-info.service.ts

```diff
@@ -21,13 +21,7 @@ export class PaymentInfoService {
     if ('currency' in dto) {
       dto.currency = await this.fiatService.getFiat(dto.currency.id);
       if (!dto.currency) throw new NotFoundException('Currency not found');
-      if ('paymentMethod' in dto && dto.paymentMethod === FiatPaymentMethod.CARD) {
-        if (!dto.currency.cardSellable) throw new BadRequestException('Currency not sellable');
-      } else if ('paymentMethod' in dto && dto.paymentMethod === FiatPaymentMethod.INSTANT) {
-        if (!dto.currency.instantSellable) throw new BadRequestException('Currency not sellable');
-      } else {
-        if (!dto.currency.sellable) throw new BadRequestException('Currency not sellable');
-      }
+      if (!dto.currency.sellable) throw new BadRequestException('Currency not sellable');
     }
```

Comment:
This check should only be made if PaymentMethod is Bank (`sellable` = `bankSellable` 😉 )